src/Sarif.Multitool.FunctionalTests/Rules/EndColumnMustNotBeLessThanStartColumnTests.cs

```diff
@@ -26,18 +25,6 @@ public void EndColumnMustNotBeLessThanStartColumn_EndColumnGreaterThanStartColum
             Verify(new EndColumnMustNotBeLessThanStartColumn(), "EndColumnGreaterThanStartColumn.sarif");
         }
 
-        [Fact(DisplayName = nameof(EndColumnMustNotBeLessThanStartColumn_EndColumnLessThanStartColumnInAnalysisTarget))]
-        public void EndColumnMustNotBeLessThanStartColumn_EndColumnLessThanStartColumnInAnalysisTarget()
-        {
-            Verify(new EndColumnMustNotBeLessThanStartColumn(), "EndColumnLessThanStartColumnInAnalysisTarget.sarif");
-        }
-
-        [Fact(DisplayName = nameof(EndColumnMustNotBeLessThanStartColumn_EndColumnLessThanStartColumnInAnalysisTargetNoEndLine))]
-        public void EndColumnMustNotBeLessThanStartColumn_EndColumnLessThanStartColumnInAnalysisTargetNoEndLine()
-        {
-            Verify(new EndColumnMustNotBeLessThanStartColumn(), "EndColumnLessThanStartColumnInAnalysisTargetNoEndLine.sarif");
-        }
```

Comment:
These two tests are obsolete because `result.AnalysisTarget` is no longer a `location` object with a `region` -- it's just a `fileLocation`.